auto_tune_vllm/cli/main.py

```
@@ -0,0 +1,856 @@
+        console.print("[bold red]Error: Must specify either --database-url or --file-path[/bold red]")
+        console.print("[yellow]Note: For file logging, you can just add 'file_path' to your study config - no setup required![/yellow]")
+        raise typer.Exit(1)
+    
+    console.print("[blue]Setting up logging infrastructure[/blue]")
+    
+    try:
+        if database_url:
+            # Setup PostgreSQL logging
+            if not verify_database_connection(database_url):
+                console.print(f"[bold red]Error: Cannot connect to database: {database_url}[/bold red]")
+                raise typer.Exit(1)
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)